shared-bindings/traceback/__init__.c

```diff
@@ -38,6 +38,75 @@
 //| ...
 //|
 
+STATIC void traceback_exception_common(mp_print_t *print, mp_obj_t value, mp_obj_t tb_obj, mp_obj_t limit_obj) {
+    if (!mp_obj_is_exception_instance(value)) {
+        mp_raise_TypeError(translate("invalid exception"));
+    }
+    mp_obj_exception_t exc = *(mp_obj_exception_t *)MP_OBJ_TO_PTR(value);
+
+    mp_int_t limit = 0;
+    bool print_tb = true;
+    if (limit_obj != mp_const_none) {
+        limit = mp_obj_get_int(limit_obj);
+        print_tb = (limit != 0);
+    }
+
+    if (tb_obj != mp_const_none && print_tb) {
+        if (!mp_obj_is_type(tb_obj, &mp_type_traceback)) {
+            mp_raise_TypeError(translate("invalid traceback"));
+        }
+        exc.traceback = MP_OBJ_TO_PTR(tb_obj);
+    } else {
+        exc.traceback = NULL;
```

Comment:
```suggestion
        exc.traceback = &mp_const_empty_traceback_obj;
```
Instead of setting traceback to `NULL`, point it to empty traceback object similar to the internal implementation.